pythran/tables.py

```diff
@@ -2814,6 +2814,12 @@ def partialsum(seq):
             return_range=lambda _: interval.Range(1, 1)
         ),
     },
+    "scipy": {
+        "special": {
+            "gammaln": ConstFunctionIntr(signature=_numpy_unary_op_float_signature),
+            "gamma": ConstFunctionIntr(signature=_numpy_unary_op_float_signature),
+        }
+    },
     "numpy": {
         "abs": ConstFunctionIntr(signature=_numpy_unary_op_signature),
```

Comment:
You still require to delete the scipy entry if scipy is not installed, which is possible. Otherwise your PR is OK, feel free to squash it!